api/utils/mappers.js

```diff
@@ -0,0 +1,125 @@
+export const airlineInfoMapper = airlines => {
```

Comment:
Exported functions from this mappers file handle converting what we get from flightStats into what our callers receive in the response object.